data/404/475/359/404475359.geojson

```diff
@@ -135,13 +139,14 @@
         }
     ],
     "wof:id":404475359,
-    "wof:lastmodified":1695877871,
+    "wof:lastmodified":1722576883,
     "wof:name":"Bancroft",
     "wof:parent_id":102081161,
     "wof:placetype":"localadmin",
-    "wof:population":68,
+    "wof:population":55,
     "wof:population_rank":1,
     "wof:repo":"whosonfirst-data-admin-us",
+    "wof:statistical_gore":1,
```

Comment:
Also curious how these were determined for some records, but not others..

Reply:
I'll upload my script to the cookbook repo... But pseudocode is if the US Census Functional Status code is in F, N, or S then it should be a statistical_gore... but the S statistical ones seem to be used in practice (mostly as counties, sometimes as localadmin) in the description of those places so I still allow them to show up in the hierarchy label, while F and N are masked.

```
if row['uscensus:funcstat'] == 'F' or row['set_uscensus_funcstat2'] == 'N' or  or row['set_uscensus_funcstat2'] == 'S':
    props['wof:statistical_gore'] = 1

if row['uscensus:funcstat'] == 'F' or row['set_uscensus_funcstat2'] == 'N':
    props['mz:hierarchy_label'] = 0
```

Reply:
Since the `wof` and `mz` properties are a calculation based on an external variable, I chose to also import `uscensus:funcstat` into the WOF documents in this PR.